server/forms.py

```diff
@@ -327,6 +327,12 @@ class CourseUpdateForm(BaseForm):
     active = BooleanField('Activate Course', default=True)
     timezone = SelectField('Course Timezone', choices=[(t, t) for t in pytz.common_timezones])
 
+class PublishScoresWithTags(BaseForm):
+    publish_tags = [tag.title() for tag in GRADE_TAGS if tag != 'private']
+    grades = SelectField('Publish Grade', choices=[(t, t) for t in publish_tags], default='Total')
```

Comment:
Instead of `tag.title` in the list comprehension above - you want it in the tuple below. 

See line 233 (https://github.com/Cal-CS-61A-Staff/ok/pull/872/files#diff-de3e8007a66bcd69728c8ddecc1f6f32L223)